python3_user_features.rst

```diff
@@ -404,3 +404,30 @@ symbols as variable names, e.g.:
 
 Use caution if you're planning to share your code, though, as it's fairly easy
 to produce illegible code this way.
+
+More useful exceptions
```

Comment:
What about "more useful exceptions when handling opening files", unless there are more examples of OSError being replaced by something easier to deal with.